skimage/segmentation/slic_superpixels.py

```diff
@@ -60,6 +144,9 @@ def slic(image, n_segments=100, compactness=10., max_iter=10, sigma=0,
         in most of the cases.
     slic_zero: bool, optional
         Run SLIC-zero, the zero-parameter mode of SLIC. [2]_
+    mask : 2D ndarray, optional
+        if provided, seed points are placed following the strategy
```

Comment:
if provided, superpixels are computed only for ``mask=True``, and seed points are placed following the strategy described in _[3], mitigating border effects at the border of the mask.